runner_manager/models/backend.py

```diff
@@ -46,3 +53,56 @@ class DockerConfig(BackendConfig):
     """Configuration for Docker backend."""
 
     base_url: str = "unix:///var/run/docker.sock"
+
+
+class GCPConfig(BackendConfig):
+    """Configuration for GCP backend."""
+
+    project_id: str
+    zone: str
+    service_account_email: str = "default"
+
+
+class GCPInstanceConfig(InstanceConfig):
```

Comment:
No success in doing this?
```suggestion
class GCPInstanceConfig(InstanceConfig, Instance):
```

Reply:
yeah.